apps/src/javalab/ControlButtons.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import color from '@cdo/apps/util/color';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+import JavalabButton from './JavalabButton';
+
+export default function ControlButtons({
+  isDarkMode,
+  isRunning,
+  isTesting,
+  toggleRun,
+  toggleTest
+}) {
+  const buttonStyles = {
+    ...styles.button.all,
+    ...(isDarkMode ? styles.button.dark : styles.button.light)
+  };
+
+  // Note: The 'noBorder' class is required on the buttons below because there are !important
+  // button styles we don't want. This class can be removed when the button:active !important
+  // border is removed from common.scss.
+  return (
+    <div
+      style={{
+        ...styles.container,
+        backgroundColor: isDarkMode ? color.black : color.white
+      }}
+    >
+      <JavalabButton
+        text={isRunning ? 'Stop' : 'Run'}
```

Comment:
Should these be internationalized? (same question for other strings)

Reply:
they weren't in i18n before, so i left them as-is, but i'll add them to the javalab/locale file